.ci.yaml

```diff
@@ -2462,6 +2622,989 @@ targets:
       - bin/
       - .ci.yaml
 
+  - name: Staging_build_linux framework_tests_widgets
```

Comment:
nit: if we're going to have these long term, maybe the bottom of the ci.yaml should be the staging builders

Reply:
Agreed. I'll move them.